pkg/manager/member/pd_member_manager.go

```diff
@@ -773,6 +785,15 @@ func getPDConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	return cm, nil
 }
 
+func clusterVersionGreaterThan4(version string) (bool, error) {
+	v, err := semver.NewVersion(version)
+	if err != nil {
+		return false, err
+	}
```

Comment:
`4.0` is the future. How about defaulting to `true` if it is not a semantic version? If people use the custom builds of old tidb versions, they can use annotation to skip TLS themself.

Reply:
done